### ISSUE TITLE
Hemolacria

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9872]
**Preferred term label:**
Hemolacria

**Synonyms**
Bloody tears, red tears

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Lacrimation abnormality HP:0000632

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Canaliculitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034803
Hemolacria
